src/main/webapp/app/routes/routes.tsx

```diff
@@ -153,11 +154,18 @@ const AppRouts = (props: {
             path={PAGE_ROUTE.TERMS}
             component={ApiAccessPageNavTab}
           />
+          <Route exact path={PAGE_ROUTE.LEVELS}>
+            <Redirect to={PAGE_ROUTE.V2} />
+          </Route>
```

Comment:
This is great. I also want to redirect user to other pages if the hash is specified. For instance if user uses `https://www.oncokb.org/levels#version=DX` I want the user to be redirected to `https://www.oncokb.org/diagnostic-levels`

Reply:
levels#version=DX currently is redirecting to therapeutic-levels, could you look into?